src/resolvers/taxonomyResolvers.ts

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { IArticleV2 } from "@ndla/types-backend/article-api";
+import { ISubjectPageData } from "@ndla/types-backend/frontpage-api";
+import { Node, TaxonomyContext } from "@ndla/types-taxonomy";
+import {
+  fetchArticle,
+  fetchChildren,
+  fetchLK20CompetenceGoalSet,
+  fetchLearningpath,
+  fetchNode,
+  fetchNodeByContentUri,
+  fetchNodeResources,
+  queryNodes,
+} from "../api";
+import {
+  GQLLearningpath,
+  GQLNode,
+  GQLNodeChildrenArgs,
+  GQLNodeCoreResourcesArgs,
+  GQLNodeSupplementaryResourcesArgs,
+  GQLQueryArticleNodeArgs,
+  GQLQueryNodeArgs,
+  GQLQueryNodeCollectionArgs,
+  GQLQueryNodeResourceArgs,
+  GQLQueryNodesArgs,
+  GQLTaxonomyContext,
+  GQLTaxonomyCrumb,
+  GQLTaxonomyEntity,
+  GQLWithArticle,
+} from "../types/schema";
+import { nodeToTaxonomyEntity } from "../utils/apiHelpers";
+import { filterMissingArticles, getArticleIdFromUrn, getLearningpathIdFromUrn } from "../utils/articleHelpers";
+
+export const Query = {
+  async node(
+    _: any,
+    { id, rootId, contextId }: GQLQueryNodeArgs,
```

Comment:
Er det rart at `contextId` kun brukes i en av stiene?

Reply:
Ikkje egentlig. Enten har du id eller så har du contextId, aldri begge. Og rootId brukes som parameter mot tax for å plukke korrekt context